src/data/modifiers.txt

```diff
@@ -2053,8 +2057,8 @@ Item	weeping willow wand	Spell Damage Percent: +100, Mysticality: +25, Experienc
 # weeping willow wand: Lets you blast your foes with tears
 Item	weighted paperclip chain	Weapon Damage: +12
 Item	western-style skinning knife	Weapon Damage: +5, Hat Drop: +10
-# whalegun: +10 Pirate Warfare Effectiveness
 Item	whalegun	Monster Level: +10, Weapon Damage: +100
+# whalegun: +10 Pirate Warfare Effectiveness
```

Comment:
so this is a bit where I presume the script is changing things on a second run (compare sawed-off blunderbuss above).

On the first run, they go in in the order they are in the description.

On second run, they go in in real / comment order.

Reply:
I haven't fixed this but I've re-run the script from scratch. I'll try to work this out when this stuff is merged